lang/python/python-awesomeversion/Makefile

```diff
@@ -0,0 +1,42 @@
+#
+# Copyright (C) 2021 CZ.NIC, z. s. p. o. (https://www.nic.cz/)
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=python-awesomeversion
+PKG_VERSION:=21.8.1
+PKG_RELEASE:=1
```

Comment:
should we use $(AUTORELEASE) here?
i'm not insisting on it, but i started switching some of my packages to it;
and i have no idea how good the mechanism is, but it's worth using it until it perfects itself;

Reply:
Yeah, sure! I will need to switch all my current maintained packages, which I maintain to it. Sometimes it is very handy.